test/unit/modules/peers.js

```diff
@@ -379,20 +379,49 @@ describe('peers', () => {
 
 		describe('networkHeight', () => {
 			before(done => {
-				randomPeers = _.range(1000).map(() => {
+				randomPeers = _.range(100).map(() => {
 					return generateRandomActivePeer();
 				});
-				peersLogicMock.list = sinonSandbox.stub().returns(randomPeers);
 				done();
 			});
 
 			const randomPeerNetworkHeight = randomPeers => {
-				const peersGroupedByHeight = _.groupBy(randomPeers, 'height');
-				const popularHeights = Object.keys(peersGroupedByHeight).map(Number);
-				return _.max(popularHeights);
+				const heights = _.countBy(randomPeers, 'height');
+				const height = Object.keys(heights).reduce((curr, next) => {
```

Comment:
It's not a good test.
It doesn't check the functionality, but rewrites and possibly repeats the mistakes introduced in code. 

Please get rid of `randomPeerNetwrokHeight` functionality. Change the generic test `should return the height of maximum number of peers at one particular height` to `should return 19 for peers being on heights: [19, 19, 19, 20, 20]`

Reply:
Bump